scripts/env.sh

```diff
@@ -2,12 +2,8 @@
 
 set -e -o pipefail
 
-# Recreate config file
-rm -rf ./env-config.js
```

Comment:
I think we still need remove the old one when we are running the script in the local env 🤔